h/queue.py

```diff
@@ -68,8 +93,8 @@ def resolve_topic(topic, namespace=None, settings=None):
 
 def includeme(config):
     config.add_request_method(
-        lambda req, t, c: get_reader(req.registry.settings, t, c),
+        lambda req, t, c: get_reader(req, t, c),
```

Comment:
Now that these functions take a request as their first argument, this can be written more straightforwardly as:

```python
config.add_request_method(get_reader, name='get_queue_reader')
```